### PR TITLE
feat: Gatsby: Additional edits of schema.org

### PR DESCRIPTION
**Pull request by redoper1**
Additional edit of schema.org as discussed in https://trello.com/c/UnkM6kna/260-schemaorg

### CODE REVIEW

**Comment by rostaklein** [comment]
I got no way to test this :( No PRs created from forks are deployed to Vercel, could you please ask for github permissions in slack and then create a regular PR for this repo please? 🙏

**Comment by redoper1** [comment]
@rostaklein > I got no way to test this in prod build :( No PRs created from forks are deployed to Vercel, could you please ask for github permissions in slack and then create a regular PR for this repo please? 🙏

Yeah, it must be confirmed manually in Vercel to deploy from fork. But you can test it here for now: https://covid-gov-cz-git-gatsby-featureschemaorg.redoper1.vercel.app/. I could ask for permissions again, but I have been told by @radkoa, that forks are ok...

File: gatsby/src/templates/custom-page/custom-page.tsx [hunk 1]
```diff
@@ -33,7 +33,10 @@ const CustomPage: React.FC<IProps> = ({ data, pageContext }) => {
         body={page.content.processed}
         title={page.title}
         langCode={pageContext.langCode}
-        breadcrumbItems={[{ title: I18n('home'), url: '/' }, page.title]}
+        breadcrumbItems={[
+          { title: I18n('home'), url: TRoute('/') },
```
Comment (vitkolos) [inline gatsby/src/templates/custom-page/custom-page.tsx:37]: Nedala by se funkce TRoute používat jenom v té jedné komponentě schema? Ať nemusí být naimportovaná a použitá všude…

Comment (redoper1) [reply gatsby/src/templates/custom-page/custom-page.tsx:37]: Zkusím otestovat odpoledne, jestli nebude dělat bordel, ale teroreticky by to mělo stačit.

Comment (redoper1) [reply gatsby/src/templates/custom-page/custom-page.tsx]: Opraveno v novém commitu

**Review by vitkolos — APPROVED** [review]

**Pull request merged**